### PR TITLE
Cross server fix

### PR DESCRIPTION
**Pull request by hiufungk**
improved server user model to have generic username, password

### CODE REVIEW

**Review by forgeno — APPROVED** [review]

File: backend/api/view/FriendsView.py [hunk 1]
```diff
@@ -83,16 +83,22 @@ def follow(request):
                 server_user = ServerUser.objects.get(host=friend_host)
                 payload = json.dumps(request.data)
                 headers = {'Content-type': 'application/json'}
-                url = server_user.host + "api/friendrequest"
-                my_cross_server_username = settings.USERNAME
-                my_cross_server_password = settings.PASSWORD
-                response = requests.post(url, data=payload, auth=(my_cross_server_username, my_cross_server_password),
+                url = "{}{}friendrequest".format(server_user.host, server_user.prefix)
+                print(payload)
```
Comment (htruong4) [inline backend/api/view/FriendsView.py]: do you need this printy boi?

File: backend/api/models.py [hunk 2]
```diff
@@ -111,6 +111,9 @@ def __str__(self):
 class ServerUser(models.Model):
     user = models.OneToOneField(User, on_delete=models.CASCADE)
     host = models.URLField()
+    prefix = models.CharField(max_length=20, blank=True)
```
Comment (htruong4) [inline backend/api/models.py:114]: what is this prefix referring to?

File: backend/api/view/FriendsView.py [hunk 3]
```diff
@@ -83,16 +83,22 @@ def follow(request):
...
+                response = requests.post(url, data=payload, auth=(server_user.send_username, server_user.send_password),
                                          headers=headers)
                 if response.status_code != 200:
-                    return Response("Cross Server Follow Request Fail", status.HTTP_400_BAD_REQUEST)
+                    # return Response("Cross Server Follow Request Fail", status.HTTP_400_BAD_REQUEST)
```
Comment (htruong4) [inline backend/api/view/FriendsView.py]: do you need this comment too?

File: backend/api/models.py (hunk 2 above)
Comment (hiufungk) [reply backend/api/models.py:114]: for example:
our team is host/api/friendrequest
bennet team is host/friendrequest

**Review by htruong4 — APPROVED** [review]

**Pull request merged**